### ISSUE TITLE
Allow forked repos

### ISSUE THREAD

**Issue by EinoHR** [VChet/StyleBase#28]
I am new to making userstyles, but I recently made one for [ecosia.org](ecosia.org) in stylus based on the [horizon vscode theme](https://github.com/jolaleye/horizon-theme-vscode).
Since that is the main theme of my machine, I forked the repo planning to add all my themes with the same colorscheme there. First I added the theme i had made for Ecosia earlier and decided to read up on how to make it shareable to everyone. Then I tried to upload it to the site, getting the error "horizon-theme repository is forked".

TLDR; Theme of my own creation inside forked repo, forked repos not allowed, what do

**Comment by FireInsight** [comment]
Update; I un-forked my fork.

**Comment by RiedleroD** [comment]
…how does one unfork a fork?

Also, I think it was done to prevent people stealing styles, but an easy solution would be to just display when a style was forked by another. Do you think there should be a separate issue for this, @VChet?

**Comment by VChet** [comment]
Thanks for feedback!
It was done to encourage people to support the original project, not its forks. I forgot that archived repositories can still be forked. This limitation will be removed and in the near future I'll be storing the parent repository to display it in the forked style.

**Comment by FireInsight** [comment]
> …how does one unfork a fork?
Just followed [this tutorial](https://stackoverflow.com/questions/38831301/how-to-un-fork-the-github-repository). I was a bit impatient but this worked.
